pkgs/applications/misc/polybar/default.nix

```
@@ -23,7 +24,7 @@
 # disable modules
 , alsaSupport   ? true,  alsaLib       ? null
 , githubSupport ? false, curl          ? null
-, mpdSupport    ? false, libmpdclient ? null
+, mpdSupport    ? false, libmpdclient  ? null
```

```suggestion
, mpdSupport    ? false, libmpdclient
```

Together with the assert is not required within the scope of nixpkgs.

---

I'll have it in a separate commit.